Resources/Computing/ARCComputingElement.py

```diff
@@ -423,6 +423,8 @@ def getJobOutput( self, jobID, localDir = None ):
 
     isItOkay = job.Retrieve( self.usercfg, arc.URL( workingDirectory ), False )
     if ( isItOkay ):
+      if not os.path.isfile(outFileName):
```

Comment:
May I recommend a more generic protection ? 

```
try:
  with open(outFileName, 'r') as outFile:
    output = outFile.read()
except IOError as e:
  return S_ERROR("Error downloading outputs", repr(e))
```

So it catches not only the fact that the path is not a file, but also if it is corrupted or whatever ?